<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>356</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>242</red>
        <green>178</green>
        <blue>55</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>42</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>242</red>
        <green>178</green>
        <blue>55</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>42</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>42</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>25</red>
        <green>42</green>
        <blue>50</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_d1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>7</y>
      <width>118</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_d2">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>6</y>
      <width>118</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_d3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>6</y>
      <width>118</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tmp">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>190</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(49, 196, 190);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_desc">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>144</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(49, 196, 190);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tmp1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>54</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 178, 55);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wind1">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>84</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 178, 55);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tmp2">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>54</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 178, 55);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wind2">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>84</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 178, 55);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tmp3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>54</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 178, 55);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wind3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>84</y>
      <width>115</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(242, 178, 55);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>331</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(186, 124, 186);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_qpix">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>120</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>279</y>
      <width>111</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(242, 178, 55)
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_wind">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>222</y>
      <width>211</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(49, 196, 190);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lne_city">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>280</y>
      <width>222</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
